src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -59,4 +59,8 @@ public String toString() {
         return '[' + tagName + ']';
     }
 
+    @Override
+    public int compareTo(Tag other) {
+        return this.tagName.compareTo(other.tagName);
+    }
```

Comment:
same here, add in the header javadoc that this class's natural ordering is alphabetical order of value